pkg/secretless/plugin/sharedobj/external_plugins.go

```diff
@@ -136,43 +136,57 @@ func ExternalPlugins(
 	for rawPluginName, rawPlugin := range rawPlugins {
 		logger.Infof("Loading plugin '%s'...", rawPluginName)
 
+		reportLoadPluginError := func(err error) {
```

Comment:
Naming nit: `reportLoadPluginError` -> `logPluginError`.  

No need to introduce a new verb `report` when we already have `log` and that's what we use everywhere.